.github/matchers/flake8-error-matcher.json

```diff
@@ -0,0 +1,17 @@
+{
+  "problemMatcher": [
+    {
+      "owner": "flake8-error",
+      "severity": "error",
+      "pattern": [
+        {
+          "regexp": "^([^:]+):(\\d+):(\\d+):\\s+([DCFNWE]\\d+\\s+.+)$",
```

Comment:
did you come up with this one yourself or did you get it from an existing flake8 annotator?

Reply:
based on https://github.com/rbialon/flake8-annotations but slightly changed (marks everything as errors rather than warnings - since the build fails on warnings and also works for documentation string warnings)